### ISSUE TITLE
Permission File - non-privileged container

### ISSUE THREAD

**Issue by cubasepp** [makandra/consul#66]
Hello,

after updating consul to 1.3.0 I cannot start my app in my non privileged container anymore.

Problem is the permission on `active_record.rb`, `errors.rb`, `guard.rb` and `dynamic_access.rb`. These files has no read permission for others.

Version 1.2.0
```

/usr/local/bundle/gems/consul-1.2.0/lib/consul/:
total 48K
drwxr-xr-x 4 root root 4.0K Feb 28 10:09 .
drwxr-xr-x 3 root root 4.0K Feb 28 10:09 ..
-rw-rw-r-- 1 root root  421 Feb 28 10:09 active_record.rb
-rw-rw-r-- 1 root root  308 Feb 28 10:09 errors.rb
-rw-rw-r-- 1 root root 2.9K Feb 28 10:09 guard.rb

/usr/local/bundle/gems/consul-1.2.0/lib/consul/power:
total 12K
drwxr-xr-x 2 root root 4.0K Feb 28 10:09 .
drwxr-xr-x 4 root root 4.0K Feb 28 10:09 ..
-rw-rw-r-- 1 root root 3.1K Feb 28 10:09 dynamic_access.rb
```

Version 1.3.0
```
/usr/local/bundle/gems/consul-1.3.0/lib/consul/:
total 52K
drwxr-xr-x 4 root root 4.0K Feb 28 10:21 .
drwxr-xr-x 3 root root 4.0K Feb 28 10:21 ..
-rw-rw---- 1 root root  421 Feb 28 10:21 active_record.rb
-rw-rw---- 1 root root  345 Feb 28 10:21 errors.rb
-rw-rw---- 1 root root 2.9K Feb 28 10:21 guard.rb

/usr/local/bundle/gems/consul-1.3.0/lib/consul/power:
total 12K
drwxr-xr-x 2 root root 4.0K Feb 28 10:21 .
drwxr-xr-x 4 root root 4.0K Feb 28 10:21 ..
-rw-rw---- 1 root root 3.1K Feb 28 10:21 dynamic_access.rb
```

Did this change happen on purpose?

**Comment by kratob** [comment]
Hi @cubasepp,

As far as I'm aware, git does not even track these kinds of file permissions (it only tracks "executable" / "not executable"). This looks like an problem in how you updated consul, not with consul itself. Maybe it's an issue with `umask`?

**Comment by dastra-mak** [comment]
I released this as 1.3.1 and unpacked the gem to make sure permissions are ok now.

**Comment by cubasepp** [comment]
Thank you very much! :)
